packages/data-api/scripts/dropAnalyticsTableIndexes.sql

```diff
@@ -1,16 +1,34 @@
 do $$
 declare
   tStartTime TIMESTAMP;
+  tAnalyticsIndexName TEXT := 'analytics_data_element_entity_date_idx';
+  tEventsIndexName TEXT := 'analytics_data_group_entity_event_date_idx';
 
 begin
   RAISE NOTICE 'Dropping analytics table indexes...';
 
-  tStartTime := clock_timestamp();
-  DROP INDEX IF EXISTS analytics_data_element_entity_date_idx;
-  RAISE NOTICE 'Dropped [data_element, entity, date] index, took %', clock_timestamp() - tStartTime;
+  IF (SELECT count(*) > 0
+    FROM pg_class c
+    WHERE c.relname = tAnalyticsIndexName 
+    AND c.relkind = 'i')
+  THEN
+    tStartTime := clock_timestamp();
+    PERFORM mv$removeIndexFromMv$Table(mv$buildAllConstants(), 'analytics_data_element_entity_date_idx');
```

Comment:
```suggestion
    PERFORM mv$removeIndexFromMv$Table(mv$buildAllConstants(), tAnalyticsIndexName);
```
Same below

Reply:
Or is this not possible because that variable is declared as a `TEXT` value or something...?

Reply:
Ah great catch, no I just forgot to do that!